files/KoreBuild/scripts/dotnet-install.ps1

```diff
@@ -71,6 +80,9 @@ param(
    [string]$Version="Latest",
    [string]$InstallDir="<auto>",
    [string]$Architecture="<auto>",
+   [ValidateSet("dotnet", "aspnetcore", IgnoreCase = $false)]
```

Comment:
Neat! Poor-man's `Enum`.

Reply:
Basically yeah, it's like an enum. Even cooler, PowerShell's tab completion will cycle through valid options only